### ISSUE TITLE
I overfit

### ISSUE THREAD

**Issue by taivop** [taivop/eth-ml#8]
I underfit...because RMSE doesn't decrease when I increase normalisation!

**Comment by taivop** [comment]
Nope. I overfit because training set error is ~640, test set error is ~730, i.e. the model doesn't generalise.

**Comment by taivop** [comment]
SOLUTION: using cross-validation, automatically remove features whose removal leads to a lower CV error!

**Comment by taivop** [comment]
Calculate baseline that includes all features. Then remove features one by one; if RMSE doesn't increase (by some epsilon amount), drop the feature.

**Comment by taivop** [comment]
Removing feature sets WITHOUT removing any of the original features:
- limit -0.1 => RMSE change -5.1
- limit 0 => RMSE change -71
